### PR TITLE
[fix][broker] Fix the wrong behaviour when set `overrideBrokerNicSpeedGbps`

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

This break was introduced by #14648 and affected unreleased 2.11.

### Modifications

- Multiply `overrideBrokerNicSpeedGbps` with the NIC number.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

- [x] `doc-not-needed`

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LinuxBrokerHostUsageImplTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+public class LinuxBrokerHostUsageImplTest {
+
+    @Test
+    public void checkOverrideBrokerNicSpeedGbps() {
+        ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
```
Comment (Technoboy-) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LinuxBrokerHostUsageImplTest.java:34]: Close this executor

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java [hunk 2]
```diff
@@ -121,10 +122,12 @@ public void calculateBrokerHostUsage() {
         this.usage = usage;
     }
 
-    private double getTotalNicLimitWithConfiguration(List<String> nics) {
+    @VisibleForTesting
+    public double getTotalNicLimitWithConfiguration(List<String> nics) {
```
Comment (congbobo184) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java:126]: it's a public method, so can delete `@VisibleForTesting`

**Review by congbobo184 — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java (hunk 2 above)
Comment (mattisonchao) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java:126]: `@VisibleForTesting ` is used to notify this method public for testing. If I change it to private the test can not work anymore.

Comment (mattisonchao) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java]: change to protected

**Review by coderzc — APPROVED** [review]

**Pull request merged**